app/src/styles.sass

```diff
@@ -164,17 +163,52 @@ $itero-mat-theme: mat-light-theme(( color: ( primary: $itero-mat-primary, accent
   right: 0
   display: flex
   flex-flow: row nowrap
-  width: 80%
+  overflow: hidden
+  width: 80vw
+  height: 150vw
   @media screen and (min-width: $breakpoint-tablet) and (max-width: $breakpoint-laptop)
-    width: 50%
+    width: 50vw
+    height: 70vw
   @media screen and (min-width: $breakpoint-laptop)
     display: none
-  .info-panel-icon #help-button
-    margin: 0.7em $small-space
+  .info-panel-content
+    position: absolute
+    z-index: 1
+    right: 0
```

Comment:
I don't see any reason to have any of these 3 lines. Why do you need them?

Reply:
These 3 lines make sure that the texts are blocked in the position that they are supposed to be. A fixed position at right and they appear in front of the main. If I delete these restrictions, the text will by displayed from the left like the photo. So I think it is helpful to keep these.
![Capture d’écran 2021-06-18 à 08 41 02](https://user-images.githubusercontent.com/83946943/122517964-ed0c6a00-d010-11eb-97fc-229741ee5b5d.png)

Reply:
This issue is due to the lines 197 to 201.